src/main/java/com/gmail/val59000mc/scenarios/scenariolisteners/MonstersIncListener.java

```diff
@@ -0,0 +1,86 @@
+package com.gmail.val59000mc.scenarios.scenariolisteners;
+
+import com.gmail.val59000mc.scenarios.ScenarioListener;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Player;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.block.data.Bisected;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.block.BlockPlaceEvent;
+import org.bukkit.event.block.BlockBreakEvent;
+import com.gmail.val59000mc.languages.Lang;
+
+public class MonstersIncListener extends ScenarioListener {
+
+    private int doorsPlaced;
```

Comment:
doorsPlaced is the same as ```doorLocs.size()``` isn't it?

Reply:
Leftover from using the map + metadata. Removed.